doc/ci.md

```diff
@@ -1,16 +1,18 @@
 # CI/CD Pipeline
 
-## Restrictions
+## Prerequisites (Schema)
 
-The following restrictions apply:
+Before we can actually execute checks or build the binaries, we need `schema.rs` and `definitions.ts`.
```

Comment:
At least for `cargo fmt` this is not true. On the contrary, it might even lead to irrelevant errors as you experienced. Linting generated code also seems to be useless (at least for us, the `diesel` people hopefully do it). For some other checks, e.g., `cargo doc` I don't know.